user-guide/introduction/shortcuts.njk

```diff
@@ -39,44 +86,50 @@ title: Shortcuts
       <td style="text-align: center;"><kbd>⌘</kbd><kbd>C</kbd></td>
     </tr>
     <tr>
-      <td>Duplicate</td>
-      <td style="text-align: center;"><kbd>Ctrl</kbd><kbd>D</kbd></td>
-      <td style="text-align: center;"><kbd>⌘</kbd><kbd>D</kbd></td>
-    </tr>
       <td>Cut</td>
       <td style="text-align: center;"><kbd>Ctrl</kbd><kbd>X</kbd></td>
       <td style="text-align: center;"><kbd>⌘</kbd><kbd>X</kbd></td>
     </tr>
+    <tr>
+      <td>Delete</td>
+      <td style="text-align: center;"><kbd>Supr</kbd>,<kbd>Delete</kbd></td>
```

Comment:
Perhaps it would be clearer `[Supr] or [Delete]` instead of `[Supr], [Delete]`? Ask to UX or designers?

Reply:
I have asked and we have decided it's better with or, so I've changed it. Good point.